src/db/sysdb.h

```diff
@@ -134,6 +134,7 @@
 #define SYSDB_CANONICAL_UPN "canonicalUserPrincipalName"
 #define SYSDB_CCACHE_FILE "ccacheFile"
 #define SYSDB_DN_FOR_MEMBER_HASH_TABLE "dnForMemberHashTable"
+#define SYSDB_SAMACCOUNTNAME "sAMAccountName"
```

Comment:
Hi,
we already read some AD specific attributes like e.g. `userAccountControl` and store it wiht an `ad` prefix as `adUserAccountControl` in the cache. Following this scheme I would suggest to call the cached attribute `adSAMAccountName`.

bye,
Sumit

Reply:
Hi Sumit,
Ok, but I'll leave the AD_ prefix in the macro anyway, right? As we have there
 `#define SYSDB_AD_USER_ACCOUNT_CONTROL "adUserAccountControl"` 
so I guess we shall keep it in the macro (i.e. `SYSDB_AD_SAMACCOUNTNAME`), too.
Let me know.
Ondrej

Reply:
Hi,

yes, I'm fine with that.

bye,
Sumit

Reply:
Fixed as per the last commit.

Reply:
Thanks.